utils/kata-manager.sh

```diff
@@ -737,6 +737,7 @@ configure_kata()
 	sudo install -o root -g root -m 0644 "$cfg_from" "$cfg_to"
 
 	sudo sed -i \
+		--follow-symlinks \
```

Comment:
Great catch @jodh-intel !
I wasnt aware myself about this behaviour myself. Reading about this, it shows that GNU sed has --follow-symlinks, while  BSD sed does not. It will be useful to atleast document this here, in case someone tries  to use this/port this to MacOS. We afterall do support macos to some extent.

We could mention something like "If you're on macOS or another BSD variant, you'll need to install GNU sed via your package manager (brew, pkg, etc)"